.babelrc

```diff
@@ -0,0 +1,3 @@
+{
+  "presets": ["@babel/preset-react"]

```

Comment:
It seems very odd that we have a `.babelrc` AND babel configs in your `webpack.config.js`. This is probably a recipe for disaster.